go/runtime/host/sgx/epid.go

```diff
@@ -52,17 +44,55 @@ func (ep *teeStateEPID) Update(ctx context.Context, sp *sgxProvisioner, conn pro
 	}
 	supportsAttestationV1 := (regParams.TEEFeatures != nil && regParams.TEEFeatures.SGX.PCS)
 
+	// XXX: Could randomize endpoint order or order it on success rates and or response times.
```

Comment:
I would just put it as a TODO (and open an issue), let's start with a simple solution for now as it is already much better than the old one.